install_config/install/advanced_install.adoc

```diff
@@ -71,9 +71,38 @@ see the
 xref:../../scaling_performance/install_practices.adoc#scaling-performance-install-best-practices[Scaling and Performance Guide].
 
 After following the instructions in the
-xref:../../install_config/install/prerequisites.adoc#install-config-install-prerequisites[Prerequisites] topic and
-deciding between the RPM and containerized methods, you can continue in this
-topic to xref:configuring-ansible[Configuring Ansible Inventory Files].
+xref:../../install_config/install/prerequisites.adoc#install-config-install-prerequisites[Prerequisites]
+topic, deciding between the RPM and containerized methods and
+xref:advanced-cloud-providers[choosing from the on-premises or cloud scenarios],
+you can continue in this topic to
+xref:configuring-ansible[Configuring Ansible Inventory Files].
+
+[[advanced-cloud-providers]]
+=== Choosing clouds over on-premises
+
+Before you begin installing {product-title} with your cloud provider, make
+sure you finished
+xref:../../install_config/install/prerequisites#prereq-cloud-provider-considerations[provisioning nodes and required infrastructure services],
+like network, storage, firewall and load balancing.  Both the provisioning and
+post-provision configuration tasks may be assisted with Ansible automation
+playbooks for the supported cloud providers. This advanced installation guide
+is happen to fulfill that purpose as well.
+
+==== OpenStack provider
+
+In order to provision and install {product-title} on OpenStack, see
+link:https://access.redhat.com/documentation/en-us/reference_architectures/2017/html-single/deploying_and_managing_red_hat_openshift_container_platform_3.6_on_red_hat_openstack_platform_10[the reference architecture for Red Hat OpenStack Platform 10],
+which is based on link:https://docs.openstack.org/heat/latest[OpenStack Heat]
+templates. See also
+xref:../../install_config/configuring_openstack#install-config-configuring-openstack[post-installation configuration]
+
+====
+[IMPORTANT]
+The reference architecture for {product-title} on Red Hat OpenStack Platform 10
+is deprecated. For OSP 13 time frame, it is being replaced with the
```

Comment:
AFAIK OCP 3.6 on OSP 10 ref. arch. is not deprecated (we've just released a few weeks ago[1]), OCP 3.4 on OSP 10 is deprecated[2]
[1] https://access.redhat.com/documentation/en-us/reference_architectures/2017/html-single/deploying_and_managing_red_hat_openshift_container_platform_3.6_on_red_hat_openstack_platform_10/
[2] https://access.redhat.com/documentation/en-us/reference_architectures/2017/html/deploying_red_hat_openshift_container_platform_3.4_on_red_hat_openstack_platform_10/

Reply:
@tomassedovic could you please comment and help to get the better wording to point OSP >10 users to the *right* docs?

Reply:
done